sdk/core/core-client-paging-rest/package.json

```diff
@@ -63,7 +63,7 @@
   "dependencies": {
     "@azure/core-paging": "^1.1.1",
     "@azure/core-rest-pipeline": "^1.1.0",
-    "@azure-rest/core-client": "1.0.0-beta.5",
+    "@azure-rest/core-client": "1.0.0-beta.6",
```

Comment:
can you update the changelog for this package as well?

Reply:
This package was not released so version is still same which has corresponding change log entry.

Reply:
Ops, I am not sure why I said this, I think I was just confused. Thanks for the clarification! I will resolve all similar comments.